### ISSUE TITLE
Hubs in 630 and 730 fields lack non-filing indicators

### ISSUE THREAD

**Issue by jodiw01** [lcnetdev/bibframe2marc-xsl#18]
When a BF description contains a Hub used as a subject, bibframe2marc places it in MARC 630, second indicator "4"

(example from MarcEdit, LCCN 2020435772)
630  \4$aBible.$pEsther$0http://id.loc.gov/resources/hubs/d3f5d64a-fee1-3fa0-2744-11ab23f5390d
(example from MarcEdit, LCCN 2024006400)
630  \4$aSesame Street (Television program)$0http://id.loc.gov/resources/hubs/a06caacc-7c62-074a-1370-ef83541accef

"4" is the correct second indicator since a thesaurus source is not specified.
The first indicator (for non-filing characters) should be "0"

**Comment by jodiw01** [comment]
This is also true when Hubs are converted to 730 fields.

bibframe2marc creates

```
</marcxml:datafield>
<marcxml:datafield tag="730" ind1="2" ind2=" ">
<marcxml:subfield code="a">Bible.</marcxml:subfield>
<marcxml:subfield code="p">Psalms.</marcxml:subfield>
<marcxml:subfield code="l">Hebrew.</marcxml:subfield>
<marcxml:subfield code="f">2022.</marcxml:subfield>
<marcxml:subfield code="1">http://id.loc.gov/resources/hubs/d59d7771-7029-6ac4-af36-29bb371247b3</marcxml:subfield>
</marcxml:datafield>
```
The first indicator of "2" should be the second indicator, and the second indicator (a blank space) should be a "0" in the first indicator.

**Comment by jodiw01** [comment]
Looks like the Hubs that create 130 and 240 fields have correctly-formatted indicators

**Comment by jodiw01** [comment]
corrected in bibframe2marc v2.6
